src/lib/Menu.js

```diff
@@ -29,7 +29,7 @@ export class Menu extends Component {
       <Nav>
         <NavMenu>
           { this.props.links.map((item, index) =>
-            <NavItem key={ index }>
+            <NavItem key={ item.name + index }>
```

Comment:
We're mixing the paddings between `{ }`. Could you enforce one style in the project?